addons/website_twitter/controllers/main.py

```diff
@@ -25,12 +26,11 @@ def get_tweets(self, limit=20):
                 return {"error": _("Please set a Twitter screen name to load favorites from, "
                                    "in the Website Settings (it does not have to be yours)")}
             return []
-        twitter_tweets = request.registry['website.twitter.tweet']
+        twitter_tweets = request.env['website.twitter.tweet']
         tweets = twitter_tweets.search_read(
```

Comment:
Any reason not to replace search_read by a regular search? The record dicts are not returned as-is.